README.md

```diff
@@ -13,19 +13,34 @@ Inline with the latest Wagtail [Long Term Support release](https://github.com/wa
 ## Getting started
 
 If you'd like to contribute to IoGT development, the best way to get started is to clone a copy of this repository and run the code on your machine.
-
-A minimal set of instructions for achieving this are:
-
 ```
-mkdir iogt-dev
-cd iogt-dev
 git clone https://github.com/unicef/iogt.git
+cd iogt
+```
+
+Create an [isolated Python environment][1], into which we will install the project's dependencies. This is highly recommended so that the requirements of this project don't interfere with any other Python environments you may have in your system, including the global Python environment.
+```
 python3 -m venv venv
```

Comment:
I usually create the `venv` directory outside of the git repository just to make it simpler when doing searches, e.g. `grep -R text_to_find *`.

However, it's okay to create it inside the repository since the `.gitignore` file will ignore the contents.  Which do you think is better for contributors (your way certainly looks tidier for the instructions!)?

Reply:
ah, I misunderstood your initial intention. I agree that the venv should be outside the project dir, though perhaps we ought not to be too prescriptive about where it should go.